apps/test-suite/tests/SQLite.ts

```diff
@@ -936,7 +972,11 @@ INSERT INTO users (name, k, j) VALUES ('Tim Duncan', 1, 23.4);
 }
 
 function addAppleAppGroupsTestSuiteAsync({ describe, expect, it, beforeEach, ...t }) {
-  const sharedContainerRoot = Object.values(Paths.appleSharedContainers)?.[0];
+  let Paths: typeof import('expo-file-system/next').Paths | null = null;
```

Comment:
`expo-file-system/next` is not available from bundle time. let's use a lazy require